### PR TITLE
Adding log level to command

### PR DESCRIPTION
**Pull request by vdesoutter**

### CODE REVIEW

File: test/integration.sh [hunk 1]
```diff
@@ -222,6 +222,17 @@ else
   exit 1
 fi
 
+echo "> test the LOG_LEVEL command"
+export CL_LOG_LEVEL=Debug
+RESULT=$($OUTPUT_DIR/cl hello)
+echo $RESULT | grep -q "Debug"
+if [ $? -eq 1 ]; then
```
Comment (gmonceyron) [inline test/integration.sh:229]:
```suggestion
if [ $? -eq 0 ]; then
```

File: cmd/root.go [hunk 2]
```diff
@@ -463,6 +463,12 @@ func secrets() []string {
 		debugFlags,
 		viper.GetString(config.DEBUG_FLAGS_KEY),
 	))
+	// append log level from configuration
+	logLevel := os.Getenv(rootCtxt.appCtx.LogLevelEnvVar())
```
Comment (bhou-crto) [inline cmd/root.go:467]: I think we should get the log level from the configuration, and expose it to the sub command

File: test/packages-src/bonjour/bonjour.bat [hunk 3]
```diff
@@ -1 +1,2 @@
 ECHO bonjour!
+ECHO $CL_LOG_LEVEL
```
Comment (gmonceyron) [inline test/packages-src/bonjour/bonjour.bat:2]: I don't think this is the way to display a variable content in in windows batch scripts

**Review by bhou-crto — CHANGES_REQUESTED** [review]

File: test/integration.sh [hunk 4]
```diff
@@ -222,6 +222,17 @@ else
   exit 1
 fi
 
+echo "> test the LOG_LEVEL command"
+export CL_LOG_LEVEL=Debug
```
Comment (gmonceyron) [inline test/integration.sh:226]: I'm not sure that this is relevant for a test, it should use file configuration, don't you think ?

**Review by gmonceyron — CHANGES_REQUESTED** [review]

**Pull request closed without merging**